[2022-06-29 20:35:06,230] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: SDG_Enrique_V0.postgreSQL_getTable 2022-06-29T20:35:03.479559+00:00 [queued]>
[2022-06-29 20:35:06,271] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: SDG_Enrique_V0.postgreSQL_getTable 2022-06-29T20:35:03.479559+00:00 [queued]>
[2022-06-29 20:35:06,273] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-06-29 20:35:06,274] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-06-29 20:35:06,276] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-06-29 20:35:06,341] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): postgreSQL_getTable> on 2022-06-29T20:35:03.479559+00:00
[2022-06-29 20:35:06,368] {standard_task_runner.py:52} INFO - Started process 12492 to run task
[2022-06-29 20:35:06,423] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'SDG_Enrique_V0', 'postgreSQL_getTable', '2022-06-29T20:35:03.479559+00:00', '--job-id', '1307', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/company_forecasting.py', '--cfg-path', '/tmp/tmp2wu2ab54', '--error-file', '/tmp/tmppycopgs2']
[2022-06-29 20:35:06,432] {standard_task_runner.py:77} INFO - Job 1307: Subtask postgreSQL_getTable
[2022-06-29 20:35:06,602] {logging_mixin.py:104} INFO - Running <TaskInstance: SDG_Enrique_V0.postgreSQL_getTable 2022-06-29T20:35:03.479559+00:00 [running]> on host 8799fce82f30
[2022-06-29 20:35:06,709] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=enrique-liebana@outlook.com
AIRFLOW_CTX_DAG_OWNER=Enrique
AIRFLOW_CTX_DAG_ID=SDG_Enrique_V0
AIRFLOW_CTX_TASK_ID=postgreSQL_getTable
AIRFLOW_CTX_EXECUTION_DATE=2022-06-29T20:35:03.479559+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-29T20:35:03.479559+00:00
[2022-06-29 20:35:06,756] {base.py:74} INFO - Using connection to: id: postgres_conection. Host: host.docker.internal, Port: 5432, Schema: data_company, Login: airflow, Password: XXXXXXXX, extra: None
[2022-06-29 20:35:21,749] {company_forecasting.py:55} INFO - Backup Company data in : /raw_data/company.csv
[2022-06-29 20:35:21,753] {python.py:118} INFO - Done. Returned value was: None
[2022-06-29 20:35:21,778] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=SDG_Enrique_V0, task_id=postgreSQL_getTable, execution_date=20220629T203503, start_date=20220629T203506, end_date=20220629T203521
[2022-06-29 20:35:21,813] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-29 20:35:21,888] {local_task_job.py:146} INFO - Task exited with return code 0
